grafana/resource_data_source.go

```diff
@@ -241,6 +247,7 @@ func makeJSONData(d *schema.ResourceData) gapi.JSONData {
 		DefaultRegion:           d.Get("json_data.0.default_region").(string),
 		CustomMetricsNamespaces: d.Get("json_data.0.custom_metrics_namespaces").(string),
 		AssumeRoleArn:           d.Get("json_data.0.assume_role_arn").(string),
+		TlsSkipVerify:           d.Get("tls_skip_verify").(bool),
```

Comment:
Could we source this from the `json_data` attribute instead since it's part of that struct?

Reply:
There are some issues around the son_data block where certain attributes are required and not needed. In the GUI for adding the resource, skip TLS verify is not bound to any other requirements. I believe TlsSkipVerify should be independant of the json_data block due to this.

Reply:
I will add some doc notes for the implementation. Thanks!